### DISCUSSION TITLE
False positive: .github

### DISCUSSION THREAD

**Discussion by jasonkarns** [sapegin/textlint-rule-terminology#74] [Bugs] [answered]
**What version of textlint are you using?**
textlint: v14.0.4

**What file type (Markdown, plain text, etc.) are you using?**
markdown (README.md)

**What did you do? Please include the actual source code causing the issue.**
Readme file contains `# .github`
Ran superlinter.

**What did you expect to happen?**
I expected it to be valid. This is the title of the special GitHub repo and cannot be changed. And accordingly, the conventional title of the readme is the name of the repo (or project)

**What actually happened? Please include the actual, raw output from textlint.**
```
Incorrect usage of the term: “github”, use “GitHub” instead  terminology
```

**Comment by sapegin** [comment, marked as answer]
This is expected, I'd recommend wrapping it in code: 

```md
# `.github`
```
